tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -12270,6 +12313,31 @@ void Sema::CheckHLSLFunctionCall(FunctionDecl *FDecl, CallExpr *TheCall,
   case hlsl::IntrinsicOp::IOP___builtin_OuterProductAccumulate:
     CheckOuterProductAccumulateCall(*this, FDecl, TheCall);
     break;
+  case hlsl::IntrinsicOp::IOP_GetAttributeAtVertex:
+    // See #hlsl-specs/issues/181. Feature is broken. For SPIR-V we want
+    // to limit the scope, and fail gracefully in some cases.
+    if (!getLangOpts().SPIRV)
+      return;
+    // This should never happen for SPIR-V. But on the DXIL side, extension can
+    // be added by inserting new intrinsics, meaning opcodes can collide with
+    // existing ones. See the ExtensionTest.EvalAttributeCollision test.
```

Comment:
This comment is stale and incorrect.  It used to precede an assert for the function name, but that was never necessary as long as the intrinsic group was checked with `IsBuiltinTable`, which it is earlier in this function.  This comment should be removed.
```suggestion
```